packages/vitest/src/node/reporters/benchmark/table/index.ts

```diff
@@ -22,6 +22,7 @@ export class TableReporter extends BaseReporter {
     if (this.isTTY) {
       this.rendererOptions.logger = this.ctx.logger
       this.rendererOptions.showHeap = this.ctx.config.logHeapUsage
+      this.rendererOptions.slowTestThreshold = this.ctx.config.slowTestThreshold
```

Comment:
For this one, it might be better to have a separate option under `benchmark.slowThreshold` or maybe just adjusting some documentation in https://vitest.dev/config/#slowtestthreshold.